docs/docs/casper/sdk.md

```diff
@@ -32,10 +32,6 @@ When working with contracts, developers can initiate interactions by calling met
 
```

Comment:
Description of Tx & Queries could be abit changed. queries does not invoke entrypoints in contract